File: app/src/main/java/com/stevenschoen/putionew/model/responses/TransfersListResponse.java
package com.stevenschoen.putionew.model.responses;

import com.stevenschoen.putionew.model.ResponseOrError;
import com.stevenschoen.putionew.model.transfers.PutioTransfer;

import java.util.List;

public class TransfersListResponse extends ResponseOrError.BasePutioResponse {
	private List<PutioTransfer> transfers;

	public List<PutioTransfer> getTransfers() {
		return transfers;
	}
}

File: app/src/main/java/com/stevenschoen/putionew/model/responses/TransfersListResponse.kt
package com.stevenschoen.putionew.model.responses

import com.stevenschoen.putionew.model.ResponseOrError
import com.stevenschoen.putionew.model.transfers.PutioTransfer

class TransfersListResponse(
    val transfers: List<PutioTransfer>
) : ResponseOrError.BasePutioResponse()
